_projects/add_repo_license.md

```diff
@@ -45,7 +45,8 @@ There are three steps for this task:
 ## Step 1
 
 Start by looking through
-[GitHub repositories without a recognised license](https://github.com/search?o=desc&q=-license%3Amit+-license%3Aapache-2.0+-license%3Absd-3-clause+-license%3Agpl+-license%3Acc+-license%3Algpl&s=stars&type=Repositories&utf8=%E2%9C%93)
+[GitHub repositories without a recognised license](https://github.com/search?o=desc&q=-license%3Amit+-license%3Aapache-
```

Comment:
rendered version does not work : https://github.com/Man-Jain/projects/blob/368416c07dc92727af63be08b5be25a8fffc5f0a/_projects/add_repo_license.md

Each Pull Request has its own version of the site which you can use to check everything.

PR 602 is https://deploy-preview-602--coala-projects.netlify.com/